tasks/install-with-package.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install Certbot.
+  package: name=certbot state=present
```

Comment:
For Debian it is certbot (with needed jessie-backports). Latest Ubuntu (16.10) also comes with certbot. Ubuntu 16.04 still calls this letsencrypt :(

Several options:
- ignore non-up-to-date distributions
- introduce a simple variable that allows to set the package name
- introduce more complexity by crafting distribution specific variable files that are included on demand
- do a quick and dirty `when` workaround for ubuntu and put letsencrypt in there

what do you think @geerlingguy ?